### DISCUSSION TITLE
Allow players to trade stuff via tribute system

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#141] [Suggestions]
I would really love to see the tribute uploads able to be used as a form of trading or selling items from one player to another via the shop. It would stimulate an economy in the server and allow players who cannot or don't want to purchase tokens to accrue tokens.

My idea is to allow players to upload items and dinos, and then be able to create a shop pack listing for it. That way a player could sell their 800 Snow Owl to another player. This would also need an option for shop packs to only allow 1 purchase and then archive the pack, to avoid 2 players buying the same item when there is only 1 in existence.

I understand if this is not doable as it sounds like a lot of work, but I think it would work out well for our community and other communities as well!

**Comment by elusive-ttv** [comment]
By creating the shop pack, I mean creating it myself! We have a ticket system in discord that would make this fairly easy for me. Also, an option to have separate items shown in the drop box would be amazing as well, if it's an option! It would make it much easier to take in items/dinos from players if they could upload all at once and it be separated on my end.

**Reply by mvaessen** [reply]
For reference, in case someone stumbles upon this later: I've created #142 to facilitate the single-purchase limit globally.

**Comment by mvaessen** [comment]
I've changed the title to take this idea a little further.

it is basically a marketplace between players that works by uploading stuff to the tribute system (and later maybe other stuff, when found useful), then have the system take the inventory, to allow them to place it on the market. Players can buy stuff on the market with tokens. Fully automated.

**Comment by LimitlessDrag0n** [comment]
That would be such a game-changer for every player. If someone wants to trade, he doesn't need to be online for that!

**Reply by mvaessen** [reply]
dont forget to upvote if you like the idea ;)

**Reply by Decollates** [reply]
I really like this premise, as long as it can have safeguards put in place to prevent players from exploiting the system (such as allowing admins to set minimums/maximums of what can and cannot be traded for), and with dealing with non-transferables like element and trophies.

**Comment by mvaessen** [comment]
@Decollates good point about restrictions. In theory GSA can detect what kinda items are in the inventory, but it may require a bit of work to sort out the different blueprint names and convert those in human-readable stuff.

**Comment by mvaessen** [comment]
Im going to reject this one as there are options available via mods
